src/Products/ProdottoVenduto.java

```diff
@@ -0,0 +1,93 @@
+package Products;
+import Enums.TipoElettronico;
+import Utility.ProdottoVendutoReader;
+import java.util.List;
+
+import java.io.FileNotFoundException;
+
+public class ProdottoVenduto {
+    private TipoElettronico tipoProdotto;
+    private int quantitaVenduta;
+    private double spesaTotale;
+    private static String stampa1 = "La media dei Smartphone venduti è:";
```

Comment:
stampa1, stampa2, stampa3 sono nomi troppo generici per delle variabile, vanno trovati dei nomi più descrittivi e possibilmente in inglese

Reply:
Ho modificato questa nomenclatura